test/quantization/core/test_workflow_module.py

```diff
@@ -106,12 +107,20 @@ def test_per_tensor_observers(self, qdtype, qscheme, reduce_range):
                     ref_scale = 0.0313725 * 255 / 127
                     ref_zero_point = -64 if qdtype is torch.qint8 else 0
             else:
-                if qscheme == torch.per_tensor_symmetric:
-                    ref_scale = 0.062745
-                    ref_zero_point = 0 if qdtype is torch.qint8 else 128
+                if qdtype == torch.qint32:
+                    if qscheme == torch.per_tensor_symmetric:
+                        ref_scale = 0.000244
```

Comment:
is this calculated from some formula?

Reply:
yes, formula from this function: https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/observer.py#L258

Reply:
formula from this function https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/observer.py#L258

Reply:
I mean should we write down the formula instead of a constant value?